### PR TITLE
Shortcuts indicators

### LINKED ISSUE #940: Add keyboard shortcut indicators
Design: https://www.figma.com/file/rBjfm2tJ2Pr9M0UNTduU9v/Joystream-Atlas?node-id=9981%3A257305 
Should be done after Introducing keyboard shortcuts to the player

### PR DESCRIPTION
**Pull request by drillprop**
Fix #940 

For now draft, I want to show animations for the designer

### CODE REVIEW

File: src/shared/components/VideoPlayer/VideoPlayer.tsx [hunk 1]
```diff
@@ -66,10 +87,30 @@ const VideoPlayerComponent: React.ForwardRefRenderFunction<HTMLVideoElement, Vid
 
   const displayPlayOverlay = playOverlayVisible && !isInBackground
 
+  useEffect(() => {
```
Comment (eldiegod) [inline src/shared/components/VideoPlayer/VideoPlayer.tsx:91]: Add comment to know what we are trying to do here for future ref

File: src/shared/components/VideoPlayer/VideoPlayer.tsx [hunk 2]
```diff
@@ -296,36 +348,137 @@ const VideoPlayerComponent: React.ForwardRefRenderFunction<HTMLVideoElement, Vid
...
+    case CustomVideojsEvents.PlayControl:
+      return {
+        icon: <SvgPlayerPlay />,
+        description: 'Play',
+        type,
+      }
+    case CustomVideojsEvents.BackwardFiveSec:
+      return {
+        icon: <SvgPlayerBackwardFiveSec />,
+        description: 'Backward 5 s',
```
Comment (eldiegod) [inline src/shared/components/VideoPlayer/VideoPlayer.tsx]:
```suggestion
        description: 'Backward 5s',
```

**Review by eldiegod — COMMENTED** [review]
Looks very solid codewise and works great, tho I found in firefox both the going backwards/forward 5 and 10 secs icons don't show for some reason

Also I'd like the animations for the indicators that disappear to be way quicker but that's something for design I guess

**Comment by drillprop** [comment]
> Looks very solid codewise and works great, tho I found in firefox both the going backwards/forward 5 and 10 secs icons don't show for some reason

Wait, what? What the hell? I'll try to find a solution for this

> Also I'd like the animations for the indicators that disappear to be way quicker but that's something for design I guess

This was agreed with a designer, so I'll leave it as it is

**Comment by drillprop** [comment]
@eldiegod should be good now, also added other player icons to avoid a similar situation in the future.

**Review by eldiegod — APPROVED** [review]
Works great !!

**Pull request merged**